resources/views/partials/footer.blade.php

```diff
@@ -1,3 +1,5 @@
+<script src="https://code.jquery.com/jquery-3.3.1.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
```

Comment:
Redundant as it already exists in head, plaz remove

Reply:
:+1: